pandas/core/shared_docs.py

```diff
@@ -114,11 +114,17 @@
 as_index : bool, default True
     For aggregated output, return object with group labels as the
     index. Only relevant for DataFrame input. as_index=False is
-    effectively "SQL-style" grouped output.
+    effectively "SQL-style" grouped output. This argument has no effect
+    on filtrations (see the `filtration user guide
+    <https://pandas.pydata.org/docs/dev/user_guide/groupby.html#filtration>`_),
+    such as ``head()``, ``tail()`` and ``nth()``.
```

Comment:
Can you add in transformations here as well?

I would describe it as "filtrations in the user guide" rather than "filteration user guide"; what do you think?

Reply:
I agree, "filtrations in the user guide" sounds more correct and more clear.